src/App.js

```diff
@@ -10,7 +10,9 @@ export class App extends Component {
   };
 
   addMovie = (movie) => {
-    // put your code here
+    this.setState((prevState) => {
+      return { movies: [...prevState.movies, movie] };
```

Comment:
не пиши в одну строку
```suggestion
      return {
        movies: [...prevState.movies, movie],
      };
```